lib/bundler/rubygems_integration.rb

```diff
@@ -514,6 +514,15 @@ def replace_entrypoints(specs)
         h
       end
 
+      Gem::Specification.send(:default_stubs, "*.gemspec").each do |stub|
```

Comment:
Using `send` here, because `default_stubs` is private. We should probably make it public in `rubygems`, or make a separate method public that calls `default_stubs` with the "*.gemspec" pattern.

Reply:
make sense. I applied it: https://github.com/rubygems/rubygems/pull/2675